server/server.js

```diff
@@ -14,16 +13,16 @@ const corsOptions = {
   optionsSuccessStatus: 200
 }
 
-// express middleware
 app.get('/customer-questions/:id', (req, res) => {
-  let productId = req.query.productId;
+  const productId = parseInt(req.params.id, 10);
   console.log(`Requesting product ${productId} from the database.`)
   db.load(productId, (err, data) => {
     if (err) {
+      console.log(err);
       res.sendStatus(404);
+    } else {
+      res.json(data);
     }
-    res.json(data);
-    res.sendStatus(200);
```

Comment:
If you want to send code and data at once, you could also try res.status(200).send(data);

Reply:
I will implement this! Thank you!